packages/ODEs/README.md

```diff
@@ -0,0 +1,168 @@
+# ODEs
+
+ODEs is a [package](https://datagrok.ai/help/develop/develop#packages) for the [Datagrok](https://datagrok.ai) platform. It provides in-browser tools for solving [intial value problem](https://en.wikipedia.org/wiki/Initial_value_problem) for systems of ordinary differential equations (ODE).
+
+* Go to **Apps** and run **EquaSleek X**
+* Enter formulas or modify template
+* Press <i class="fas fa-play"></i> **Run** button on the top panel
+
+The solver requires ODEs specification in the declarative form.
+
+## Quick start
+
+Datagrok ensures intuitive tools for the rapid solving ODEs.
+
+* Run the solver: 
+  * Go to **Apps** and run **EquaSleek X**
+  * Press <i class="fas fa-play"></i> **Run** button on the top panel
+* Modify a template:
+  * Edit formulas or add new ones
+  * Click **F5** or <i class="fas fa-play"></i> **Run** button
+* Use an advanced template:
+  * Rigth click and select **Templates > Advanced...**
+  * Modify formulas and click **F5**
+* Save formulas in a local file:
+  * Rigth click and select **Save...**
+  * Find the file in Downloads
+* Load equations from a local file:
+  * Rigth click and select **Load...**
+  * Select a file with formulas
+
+## Creating project
+
+The platform provides the creation of an ODEs project in a few clicks.
```

Comment:
I propose to use "task" instead of "project". What do you think?
"equations" has been already used in control statements.
Also, I've removed multiple hierarchy.